### ISSUE TITLE
Linux Assets of the v0.2.0

### ISSUE THREAD

**Issue by mr-ili** [orange-cloudfoundry/credhub_exporter#14]
Hello there

First of all, I would like to say thank you for your job :)
I have a few questions about assets from the last release:

* Could you give any tips about this `https://github.com/orange-cloudfoundry/credhub_exporter/releases/download/v0.2.0/credhub_exporter_linux_amd64` asset? Is it a binary? I am confused because you usually used another schema of the naming assets.
* Would you plan to continue to use the new schema to name assets? Might I ask you to re-upload assets by using old name schema?

I am looking forward to your answers:)

**Comment by psycofdj** [comment]
Hi,

I've switched to [goreleaser](https://github.com/orange-cloudfoundry/credhub_exporter/blob/master/.goreleaser.yml) for this version which produces the release assets. The format is now stable and I wasn't planning to re-upload with the old schema.

After verification, there is indeed a problem with the linux binary which is actually a darwin build (`Mach-O 64-bit x86_64 executable`).  I don't know what happened and I'll fix this right away and re-upload the binaires. The `.zip` version of the linux build is correct.

Sorry for the inconvenience

**Comment by psycofdj** [comment]
The binary as been re-upload and is not matching sha256 if file `checksums.txt`

**Comment by psycofdj** [comment]
Is the new format a possible solution in your use-case as long as it remain stable from now on ?

**Comment by mr-ili** [comment]
Thank you for the fast response and for updating the binary. :)

How we use those assets: We have a pipeline which is downloading tar.gz file from URL that has a template like `https://github.com/orange-cloudfoundry/credhub_exporter/releases/download/v__version__/credhub_exporter-__version__.linux-amd64.tar.gz` than we extract it.
So now I need to change the template - there is not a lot of work.
Then I need to fix a script for this(I mean to skip unpacking step) - and it might be painful because other binaries came to us like a tar file).

To sum up, for me it will be the best solution to continue work with tar files but if you want to continue work without the tar files,  we have to rewrite some part of our pipeline. Please, let me know if you are not planning to use tar files:)

**Comment by psycofdj** [comment]
Go for the pipeline modifications, we'll stick to the current format
